### ISSUE TITLE
Implement keyboard "space" and "shift+space" scrolling in data-tables and other views

### ISSUE THREAD

**Issue by marcinbauer-ohf** [home-assistant/home-assistant-polymer#51360]
### Description

👋 
Currently, users viewing a data table must use the mouse scroll wheel or scrollbar to navigate through rows. There is no keyboard shortcut to scroll the table content.

Add support for Space and Shift+Space to scroll data tables down and up respectively, consistent with standard browser and OS scrolling conventions. This should work whenever the user has interacted with a table, and should not interfere with typing in the search input or other interactive fields.

Pages that currently doesn't support this shortcut:
- All data-tables (Devices & Services, Dashboards, Voice->Expose.. etc.)
- Integrations

**Comment by timmo001** [comment]
This is likely due to lack of focus on the data table on load. Also should be able to use page up/down and up/down navigation

**Comment by timmo001** [comment]
Last PR will fully complete this
